src/app/core/components/manage-tags/manage-tags.component.ts

```diff
@@ -0,0 +1,71 @@
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+import { TagVO } from '@models';
+import { ApiService } from '@shared/services/api/api.service';
+import { PromptService } from '@shared/services/prompt/prompt.service';
+
+@Component({
+  selector: 'pr-manage-tags',
+  templateUrl: './manage-tags.component.html',
+  styleUrls: ['./manage-tags.component.scss']
+})
+export class ManageTagsComponent implements OnInit {
+  @Input() tags: TagVO[] = [];
+  @Output() refreshTags = new EventEmitter<void>();
+
+  public editingTagIds: number[] = [];
+  public filter: string = '';
+  public originalTagNames: string[] = [];
+
+  constructor(
+    private api: ApiService,
+    private prompt: PromptService
+  ) { }
+
+  ngOnInit(): void {
+  }
+
+  public getFilteredTags(): TagVO[] {
+    return this.tags.filter((t) =>
+      t.name.toLocaleLowerCase().includes(
+        this.filter.trim().toLocaleLowerCase()
+      )
+    ).sort((a, b) => a.name.localeCompare(b.name));
+  }
```

Comment:
That summary sounds right to me! And the code appears to be pretty idiomatic. What would you suggest for a docstring?

Reply:
It took me a bit to grok what was happening because this uses "filter" instead of "search" in the method name (which I believe is because it's using a pattern called "filter" and is correct!) so I would appreciate a hint that points the casual reader in the right direction.  "Search tags for user-provided string" or something.  BUT that's just me, no need to block on it.